internal/handlers/pg/msg_saslstart.go

```diff
@@ -75,10 +75,13 @@ func (h *Handler) MsgSASLStart(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 		return nil, lazyerrors.Error(err)
 	}
 
+	var emptyPayload types.Binary
 	var reply wire.OpMsg
 	must.NoError(reply.SetSections(wire.OpMsgSection{
 		Documents: []*types.Document{must.NotFail(types.NewDocument(
+			"conversationId", int32(1),
 			"done", true,
+			"payload", emptyPayload,
```

Comment:
```suggestion
			"payload", types.Binary{},
```

Reply:
Will that be nil? Just wondering, I assume so :)

Reply:
`nil` is not a valid type on our code base; we need non-nil value there with a right type

Reply:
Ah, but yes, `(types.Binary)(nil)` would work. Ok, nevermind, I will get back to my work… 😆

Reply:
@b1ron sorry, please revert this change, the original code was right